how_to_submit_a_pr_-_quick_guide.md

```diff
@@ -0,0 +1,83 @@
+# How to submit a Pull Request - Quick guide
+
+
+## Objective
+
+After reading, you will know how to submit Pull Request (PR) for code review.
+
+## Prerequisite
+
+* I Understand the project goal and what is its main purpose.
+* I know who is the team lead and account manager.
+
+
+
+## The Flow
+
+1. **Open your task** on Github: Go to `Project` tab, click on `Weekly Workplan`, under your name you will see all your tasks in the project.
```

Comment:
Can we include how the tasks were determined/assigned. Something like: "Every Thursday, the Account Managers review the project, pull out the most urgent issues, and assign the developers they think are best fit."  I don't know if this is actually how it happens. ;)  This will help developers understand where and how they fit into the project.